user-console/src/main/java/org/pentaho/mantle/client/admin/UserDialog.java

```diff
@@ -148,13 +151,28 @@ public void onResponseReceived( Request request, Response response ) {
         String userName = nameTextBox.getText();
         String password = passwordTextBox.getText();
         String reservedCharacters = response.getText();
+        String allowedChars = "a-z A-Z 0-9 _ . , : ; < > | ! @ # $ % ^ & * ( ) [ ]";
```

Comment:
This should be a static final and should be declared near the "regex variables" in line 52/53.